From 7bfe3bfb4cd7d1db43b1baeefc0cfbf2f648a092 Mon Sep 17 00:00:00 2001
From: Xiaofei Tan <tanxiaofei@huawei.com>
Date: Tue, 23 Sep 2025 11:58:03 +0800
Subject: [PATCH 9/9] sysSentry: Add MulanV2 License statement

driver inclusion
category: feature
bugzilla: https://gitee.com/src-openeuler/sysSentry/issues/ID1UOY
CVE: NA

----------------------------------------------------------------------

Add MulanV2 license statement to the header of each file.

Signed-off-by: Xiaofei Tan <tanxiaofei@huawei.com>
Signed-off-by: Qizhi Zhang <zhangqizhi3@h-partners.com>
---
 src/sentryPlugins/soc_ring_sentry/soc_ring_sentry.c | 12 +++++++++++-
 src/sentryPlugins/soc_ring_sentry/soc_ring_sentry.h | 12 +++++++++++-
 src/sentryPlugins/soc_ring_sentry/tc_ring_one.c     | 12 +++++++++++-
 src/sentryPlugins/soc_ring_sentry/tc_ring_one.h     | 12 +++++++++++-
 4 files changed, 44 insertions(+), 4 deletions(-)

diff --git a/src/sentryPlugins/soc_ring_sentry/soc_ring_sentry.c b/src/sentryPlugins/soc_ring_sentry/soc_ring_sentry.c
index df195ae..d5e615c 100644
--- a/src/sentryPlugins/soc_ring_sentry/soc_ring_sentry.c
+++ b/src/sentryPlugins/soc_ring_sentry/soc_ring_sentry.c
@@ -1,5 +1,15 @@
 /*
- * Copyright (c) Huawei Technologies Co., Ltd. 2025. All rights reserved.
+ * Copyright (c) 2025 Huawei Technologies Co., Ltd.
+ * sysSentry is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at:
+ *          http://license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ *
+ * See the Mulan PSL v2 for more details.
+ *
  * Description: SOC Ring sentry main program
  * Author: Yihang Li
  * Create: 2025-7-10
diff --git a/src/sentryPlugins/soc_ring_sentry/soc_ring_sentry.h b/src/sentryPlugins/soc_ring_sentry/soc_ring_sentry.h
index 0566496..34fb5ec 100644
--- a/src/sentryPlugins/soc_ring_sentry/soc_ring_sentry.h
+++ b/src/sentryPlugins/soc_ring_sentry/soc_ring_sentry.h
@@ -1,5 +1,15 @@
 /*
- * Copyright (c) Huawei Technologies Co., Ltd. 2025. All rights reserved.
+ * Copyright (c) 2025 Huawei Technologies Co., Ltd.
+ * sysSentry is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at:
+ *          http://license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ *
+ * See the Mulan PSL v2 for more details.
+ *
  * Description: SOC Ring sentry main header
  * Author: Yihang Li
  * Create: 2025-7-10
diff --git a/src/sentryPlugins/soc_ring_sentry/tc_ring_one.c b/src/sentryPlugins/soc_ring_sentry/tc_ring_one.c
index 3dda5e0..bae7ada 100755
--- a/src/sentryPlugins/soc_ring_sentry/tc_ring_one.c
+++ b/src/sentryPlugins/soc_ring_sentry/tc_ring_one.c
@@ -1,5 +1,15 @@
 /*
- * Copyright (c) Huawei Technologies Co., Ltd. 2025. All rights reserved.
+ * Copyright (c) 2025 Huawei Technologies Co., Ltd.
+ * sysSentry is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at:
+ *          http://license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ *
+ * See the Mulan PSL v2 for more details.
+ *
  * Description: tc ring testcase program
  * Author: lizixian
  * Create: 2025-7-10
diff --git a/src/sentryPlugins/soc_ring_sentry/tc_ring_one.h b/src/sentryPlugins/soc_ring_sentry/tc_ring_one.h
index 5e93a56..126f9cb 100755
--- a/src/sentryPlugins/soc_ring_sentry/tc_ring_one.h
+++ b/src/sentryPlugins/soc_ring_sentry/tc_ring_one.h
@@ -1,5 +1,15 @@
 /*
- * Copyright (c) Huawei Technologies Co., Ltd. 2025. All rights reserved.
+ * Copyright (c) 2025 Huawei Technologies Co., Ltd.
+ * sysSentry is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at:
+ *          http://license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ *
+ * See the Mulan PSL v2 for more details.
+ *
  * Description: tc ring testcase header
  * Author: lizixian
  * Create: 2025-7-10
-- 
2.33.0

